src/pages/addEvent.vue

```diff
@@ -0,0 +1,60 @@
+<template>
```

Comment:
This page should be linked from events.vue page,
dont make a separate nav item for this

Reply:
Help Needed

Reply:
I will implement routers 
That will make things very easy.
For now remove all changes except the addEvents file. When I make the changes, rebase and then work on that